app/models/version.rb

```diff
@@ -393,6 +393,10 @@ def manifest
     rubygem.version_manifest(number, platformed? ? platform : nil)
   end
 
+  def contents_payload
+    manifest.checksums.transform_values { |checksum| { "sha256" => checksum } }
```

Comment:
Is it good idea to rely on `checksums` being sha256 in here? Wouldn't make sense to get this info from manifest (even hardcoded in there) for now?

Reply:
Also please could it be possible to add test?

Reply:
yes, I should do that. Thanks for noticing.